scripts/freebsd/update.sh

```diff
@@ -19,7 +19,7 @@ fi
 
 # Update FreeBSD
 # NOTE: the install action fails if there are no updates so || true it
-env PAGER=/bin/cat $freebsd_update fetch;
+env PAGER=/bin/cat $freebsd_update fetch || true;
```

Comment:
11.0 fails on fetch. I guess it's because there is no mirrors yet. Didn't have time to dig into it.